ssz/sedes/container.py

```diff
@@ -162,5 +167,35 @@ def get_hash_tree_root(self, value: Tuple[Any, ...]) -> bytes:
         )
         return merkleize(merkle_leaves)
 
+    def get_hash_tree_root_and_leaves(self, value: Tuple[Any, ...], cache) -> bytes:
```

Comment:
Return type here needs to be updated to indicate it returns the `cache` and the `cache` needs a type hint (this should be enforced by CI....)